ansible/roles/erwin-idm/templates/deployment.yml.j2

```
@@ -0,0 +1,67 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: erwinidm-deployment
+  namespace: {{ NAMESPACE }}
+  labels:
+    app: erwinidm
+spec:
+  replicas: {{ ERWINIDM_REPLICAS|default(1, true) }}
+  strategy:
+    type: Recreate
```

It's better to use the strategy RollingUpdate for this service.
If you change it plese delete this service first at the develop enviorments
```suggestion
  strategy:
    type: RollingUpdate
    rollingUpdate:
      maxSurge: 1
      #maxUnavailable: 1
```

---

done